index.js

```diff
@@ -89,7 +89,7 @@
   if (typeof module !== 'undefined') {
     module.exports = format;
   } else if (typeof define === 'function' && define.amd) {
-    define(format);
+    define(function () { return format; });
```

Comment:
If you delete the space between `function` and `()`, `make lint` should pass. Please `git commit --amend && git push origin master --force`. :)